android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/components/datetimepicker/RNDatePickerDialogFragment.java

```diff
@@ -7,8 +7,6 @@
 
 package versioned.host.exp.exponent.modules.api.components.datetimepicker;
 
-import host.exp.expoview.R;
```

Comment:
Are you sure we can remove that? I see some references to `R.style` below in this file

Reply:
I'll validate on Android, I just updated from source with the command. Did not change anything else.

Reply:
Since this package is still using the old vendoring script, I'll be fine if you just fix this manually. This kind of imports will become unnecessary when we switch to the new script (should probably do that in the next release 😅)

Reply:
Yep, you are right, it's missing in two files. Patching manually now!

Reply:
Done!